### DISCUSSION TITLE
Difference between MQTT and HASS light groups

### DISCUSSION THREAD

**Discussion by TMV-NL** [Koenkk/zigbee2mqtt#11131] [General]
Hi,
I've grouped 12 recessed lights together with default HASS groups. But I notice that when commanding a change in scene, or switching off, not all lights respond properly.
I understand that MQTT addresses light groups differently than HASS. Would this possibly solve my issue?
And can I use the MQTT groups also in lovelace dashboards?
Thanks

**Comment by Koenkk** [comment]
Yes,
- HA groups: will send a command to each light (so 12 lights = 12 commands)
- Zigbee2MQTT goups: will send 1 broadcast messages which are handled by all lights that are in the group

**Comment by TMV-NL** [comment]
Thanks
In order to use MQTT groups in the dashboard, should I then publish messages to an MQTT topic?

**Reply by Koenkk** [reply]
Frontend doesn't allow to control groups, but groups will be discoverd to HA

**Reply by robertalexa** [reply]
I can confirm the above.

Here is a z2m group into HA
![image](https://user-images.githubusercontent.com/9019306/152536092-70de80ee-457d-4344-bdd0-1277306d5e26.png)

And the 2 lights that make it up are also exposed individually
![image](https://user-images.githubusercontent.com/9019306/152536141-ab49fcff-1b93-4919-82cc-f9f4326c9506.png)

**Reply by atv2016** [reply]
so does service.lights include the zigbee created group ?

**Reply by TMV-NL** [reply]
So, if I make a MQTT group of 4 lights and in the UI a scenery and apply it via an UI automation to the MQTT group, it will be send as 1 message? Or do I need to create a MQTT scenery for this?
